src/App.js

```diff
@@ -1,29 +1,11 @@
-import logo from './logo.svg';
+import react from 'react';
 import './App.css';
+import Calculator from './components/Calculator';
```

Comment:
- [ ] - [ ] it would be better if `MIT` link on your `README` file is still not working please make sure to make it work as expected when I click on it I should see the License

Reply:
@youmari Awaiting Approval. Thanks.